plugins/python-build/test/pyenv.bats

```diff
@@ -140,7 +140,8 @@ OUT
   stub_python_build 'echo ERROR >&2 && exit 2'
   stub_python_build "--definitions : echo 2.6.9 2.7.9-rc1 2.7.9-rc2 3.4.2 | tr ' ' $'\\n'"
 
-  run pyenv-install 2.7.9
+  mkdir "$BATS_TEST_TMPDIR/.git"
+  PYENV_INSTALL_PATH="$BATS_TEST_TMPDIR" run pyenv-install 2.7.9
```

Comment:
Not sure why adaptations like this are needed (same elsewhere). Setting `PYENV_INSTALL_PATH` in the common `setup()` should be all that's needed. If it's not, something's probably not right.